stream_alert_cli/terraform/streamalert.py

```diff
@@ -44,6 +44,8 @@ def generate_stream_alert(cluster_name, cluster_dict, config):
     """
     account = config['global']['account']
     modules = config['clusters'][cluster_name]['modules']
+    vpc_subnets = config['clusters'][cluster_name].get("vpc_subnets", "")
```

Comment:
maybe I'm misunderstanding what the value of these will/should be, but I think the default value here should be an empty list `[]` not empty string, otherwise the tf will fail since it's expecting a list: https://github.com/airbnb/streamalert/pull/819/files#diff-ef86a913d13a64356b590676f0fbce8eR73

Reply:
Can we use single quotes to be consistent? `"vpc_subnets"` --> `'vpc_subnets'`

Reply:
i'm not sure -- [docs](https://www.terraform.io/docs/providers/aws/r/lambda_function.html#vpc_config) say:
 `NOTE: if both subnet_ids and security_group_ids are empty then vpc_config is considered to be empty or unset.`